src/app/components/listing/details/surveillance/nonconformity/edit.component.js

```diff
@@ -52,26 +51,29 @@ export const SurveillanceNonconformityEditComponent = {
       if (this.nonconformity.capMustCompleteDate) {
         this.nonconformity.capMustCompleteDateObject = new Date(this.nonconformity.capMustCompleteDate);
       }
+      if (this.nonconformity.nonconformityCloseDate) {
```

Comment:
Should this have the capitalized conformity?

```suggestion
      if (this.nonconformity.nonConformityCloseDate) {
```